src/components/touchableOpacity/index.tsx

```diff
@@ -79,14 +79,22 @@ class TouchableOpacity extends PureComponent<Props, {active: boolean}> {
 
   onPressIn = (...args: any) => {
     this.setState({active: true});
-    //@ts-expect-error
-    this.props.onPressIn?.(...args);
+    if (this.props?.customValue) {
```

Comment:
Consider using a ternary for this condition it will make the code a bit short and cleaner (: